block/polka.go

```diff
@@ -63,16 +101,36 @@ type Polka struct {
 	Signatories sig.Signatories
 }
 
+// Equal checks that two Polka are functionally equivalent
+func (polka Polka) Equal(other Polka) bool {
+	if other.Block == nil {
+		return polka.Block == other.Block
+	}
+	return polka.Block.Equal(*other.Block) &&
+		polka.Round == other.Round &&
+		polka.Height == other.Height &&
+		polka.Signatures.Equal(other.Signatures) &&
+		polka.Signatories.Equal(other.Signatories)
+}
+
 func (polka Polka) String() string {
 	blockHeader := "Nil"
 	if polka.Block != nil {
 		blockHeader = base64.StdEncoding.EncodeToString(polka.Block.Header[:])
 	}
-	return fmt.Sprintf("Polka(Block=%s,Round=%d,Height=%d)", blockHeader, polka.Round, polka.Height)
+	return fmt.Sprintf("Polka(Block=%s,Round=%d,Height=%d)",
```

Comment:
Why are all these lines being wrapped? Only comments should get line wrapped. Everything else is exactly as `gofmt` provides.